packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts

```diff
@@ -115,19 +116,37 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
     });
 
     execution.stderrSubject.subscribe(async data => {
-      if (data && data.toString().includes('Server start up failed')) {
-        this.handleErrors(cancellationToken, serverHandler, serverStarted, 1);
-        progress.complete();
+      if (!printedError && data) {
+        let errorCode = -1;
+        if (data.toString().includes('Server start up failed')) {
+          errorCode = 1;
+        }
+        if (data.toString().includes('EADDRINUSE: address already in use')) {
```

Comment:
I would scope this down to only checking for `EADDRINUSE` since that's the error code and the message `address already in use` could change.